src/modules/uv-shared-module/BaseExtension.ts

```diff
@@ -889,8 +889,12 @@ export class BaseExtension implements IExtension {
         return null;
     }   
 
-    getIIIFShareUrl(): string {
-        return this.helper.iiifResourceUri + "?manifest=" + this.helper.iiifResourceUri;
+    getIIIFShareUrl(shareManifests: boolean = false): string {
+        if (shareManifests) {
```

Comment:
I wonder if this would be a little simpler as:

<pre>
var manifestUri = shareManifests ? this.helper.manifest.id : this.helper.iiifResourceUri;
return `${manifestUri}?manifest=${manifestUri}`;
</pre>

I was momentarily confused by the fact that the same concatenation logic is being applied to different variables using different syntaxes in the if..else ... which led me to thinking of this minor optimization. :-)

Reply:
(And, of course, I'm just writing Javascript there... it should be typed properly since this is TypeScript... but you get the idea).